k8s/configmaps.go

```diff
@@ -16,7 +18,23 @@ func GetConfigMap(ctx context.Context, name, namespaceOverride string) (*corev1.
 	return client.CoreV1().ConfigMaps(namespace).Get(ctx, name, metav1.GetOptions{})
 }
 
-func ListConfigMapsNames(ctx context.Context, namespaceOverride string) (names []string, err error) {
+// ListConfigMapsNamesIfConnected lists names of ConfigMaps present and the current k8s context
+// returns empty list, if not connected to any cluster
+func ListConfigMapsNamesIfConnected(ctx context.Context, namespaceOverride string) (names []string, err error) {
+	names, err = listConfigMapsNames(ctx, namespaceOverride)
+	if err != nil {
+		var dnsErr *net.DNSError
```

Comment:
Why only DNS error? What about when we are not logged in at all?

Reply:
Right, fixing.